internal/unionstore/union_store.go

```diff
@@ -236,6 +237,8 @@ type MemBuffer interface {
 	Flush(force bool) (bool, error)
 	// FlushWait waits for the flushing task done and return error.
 	FlushWait() error
+	// FlushWaitDuration returns the total duration spent on FlushWait()
+	FlushWaitDuration() time.Duration
```

Comment:
In case we may need more execution details like `flushDuration`, do we need to make the interface something like `FlushDetails`?

Reply:
That looks reasonable